cmd/lookoutd/serve.go

```diff
@@ -87,7 +87,7 @@ func (c *ServeCommand) Execute(args []string) error {
 	}()
 
 	go func() {
-		enqErrCh <- c.runEventEnqueuer(ctx, qOpt, watcher)
+		enqErrCh <- c.runEventEnqueuer(ctx, qOpt, watcher, false)
```

Comment:
why don't you want to apply cache here too?

Reply:
I figured we could simplify and skip this step for the memory queue.
But, at the same time, it could save us from going to the DB constantly to check for already processed events, so... should we enable it too?

Reply:
Up to you, but I don't like having lots of log with `skipping` that we have right now. The cache will remove it and will stop calling DB all the time. We may also remove this flag from `runEventEnqueuer` and always use decorator inside (less code!).

Reply:
OK, done in fa55b9a